### PR TITLE
DOC-1737: RC: Connection Wizard

### PR DESCRIPTION
**Pull request by cmilesb**
Staging link: https://docs.redis.com/staging/preview-rc-connect/rc/rc-quickstart/

### CODE REVIEW

**Comment by cmilesb** [comment]
Opening PR for merge on GA date for connection Wizard.

File: content/rc/rc-quickstart.md [hunk 1]
```diff
@@ -64,30 +64,27 @@ When your new database becomes active, you're ready to connect to it.
...
-- The hostname for your database
-- The port number
-- The database password
+The **Security** section contains your **Default user password**.  By default, this is masked.  Select the eye icon to show or hide the password.    
 
-These are displayed in the **Configuration** tab.  
+{{<image filename="images/rc/database-fixed-configuration-security.png" width="75%" alt="The Security section of the Configuration tab of the database details page." >}}{{< /image >}}
 
-- In the **General** section, the **Public endpoint** setting shows the hostname for your database and the port number.
+Once you have the username and password, select the **Connect** button to open the connection wizard.
```
Comment (kaitlynmichael) [inline content/rc/rc-quickstart.md:73]:
```suggestion
Once you have the username and password, select **Connect** to open the connection wizard.
```

File: content/rc/rc-quickstart.md [hunk 2]
```diff
@@ -64,30 +64,27 @@ When your new database becomes active, you're ready to connect to it.
...
 
-- The **Security** section contains your **Default user password**.  By default, this is masked.  Select the eye icon to show or hide the password.    
+![Connect button](/images/rc/connection-wizard-button.png#no-click "Connect button.")
 
-    {{<image filename="images/rc/database-fixed-configuration-security.png" width="75%" alt="The Security section of the Configuration tab of the database details page." >}}{{< /image >}}
-
-
-Once you have the connection details, you can connect in a variety of ways, including:
+The connection wizard shows how to connect to your database with the following methods:
```
Comment (kaitlynmichael) [inline content/rc/rc-quickstart.md:77]:
```suggestion
The connection wizard provides the following database connection methods:
```

File: content/rc/rc-quickstart.md [hunk 3]
```diff
@@ -64,30 +64,27 @@ When your new database becomes active, you're ready to connect to it.
...
 
 - Using the `redis-cli` utility
```
Comment (kaitlynmichael) [inline content/rc/rc-quickstart.md:79]:
```suggestion
- redis-cli` utility
```

File: content/rc/rc-quickstart.md [hunk 4]
```diff
@@ -64,30 +64,27 @@ When your new database becomes active, you're ready to connect to it.
...
 
-- Using a [connection client](https://redis.io/clients) for your preferred programming language
+- Using a [Redis client](https://redis.io/clients) for your preferred programming language
```
Comment (kaitlynmichael) [inline content/rc/rc-quickstart.md:81]:
```suggestion
- [Redis client](https://redis.io/clients) for your preferred programming language
```

File: content/rc/rc-quickstart.md [hunk 5]
```diff
@@ -64,30 +64,27 @@ When your new database becomes active, you're ready to connect to it.
...
 
-Here's an example of each.
+- Using [RedisInsight](https://redis.com/redis-enterprise/redis-insight/)
```
Comment (kaitlynmichael) [inline content/rc/rc-quickstart.md:83]:
```suggestion
- [RedisInsight](https://redis.com/redis-enterprise/redis-insight/)
```

File: content/rc/rc-quickstart.md [hunk 6]
```diff
@@ -129,36 +124,76 @@ Run the following commands to create a `redis` Docker container and connect to y
...
+- Python using [redis-py](https://github.com/redis/redis-py#redis-py)
+- Java using [Jedis](https://github.com/redis/jedis#jedis)
+
+See the [client list](https://redis.io/docs/clients/) to view all Redis clients by language.
 
-Different programming languages use different clients to interact with Redis databases.
+#### Code example (Python)
 
-Here's how to connect to your database using the `redis-py` library for Python.
+Here's how to connect to your database using the [redis-py](https://github.com/redis/redis-py#redis-py) library for Python.
```
Comment (kaitlynmichael) [inline content/rc/rc-quickstart.md:144]:
```suggestion
To connect to your database using the [redis-py](https://github.com/redis/redis-py#redis-py) library for Python:
```

File: content/rc/rc-quickstart.md [hunk 7]
```diff
@@ -129,36 +124,76 @@ Run the following commands to create a `redis` Docker container and connect to y
...
+
+RedisInsight is a free Redis GUI that is available for MacOS, Windows, and Linux.
+
+1. In the connection wizard, under **Redis Client**, select your operating system from the **Download RedisInsight** menu.
+
+1. Select **Download** to download RedisInsight.
+
+1. [Install RedisInsight]({{< relref "/ri/installing/install-redis-desktop" >}}).
+
+1. After you install and open RedisInsight, select **Add Redis Database**.
```
Comment (kaitlynmichael) [inline content/rc/rc-quickstart.md:188]:
```suggestion
1. Open RedisInsight and select **Add Redis Database**.
```

File: content/rc/rc-quickstart.md [hunk 8]
```diff
@@ -129,36 +124,76 @@ Run the following commands to create a `redis` Docker container and connect to y
...
+
+1. In the connection wizard, under **RedisInsight Desktop**, select **Copy** to copy the connection information.
+
+1. In RedisInsight, enter the copied connection information into the **Host** field. RedisInsight automatically populates the rest of the fields needed to connect to the database as the default user.
```
Comment (kaitlynmichael) [inline content/rc/rc-quickstart.md:192]:
```suggestion
1. In RedisInsight, paste the connection information into the **Host** field. 
  RedisInsight automatically populates the rest of the fields needed to connect to the database as the default user.
```
Add the second sentence as an indented line below the step.

File: content/rc/rc-quickstart.md [hunk 9]
```diff
@@ -129,36 +124,76 @@ Run the following commands to create a `redis` Docker container and connect to y
...
-    sudo pip install redis
+    $ sudo pip install redis
     ```
 
-2.  The specific syntax varies according to the client:
+2.  In the connection wizard, under **Redis Client**, select Python from the **Select your client** menu.
+
+3. Select **Copy** to copy the connection code for your database.
+
+4. Add the copied code to your program and replace `<username>` and `<password>` with your username and password.
```
Comment (kaitlynmichael) [inline content/rc/rc-quickstart.md:156]:
```suggestion
4. Paste the code to your program and replace `<username>` and `<password>` with your username and password.
```

**Review by kaitlynmichael — CHANGES_REQUESTED** [review]
Left some suggestions for tightening up wording.

**Review by kaitlynmichael — APPROVED** [review]

**Pull request merged**